### PR TITLE
Fixed broken use statements

### PR DESCRIPTION
**Pull request by Nyholm**
Inspired by https://github.com/symfony/symfony-docs/pull/14429 I decided to fix the other broken statements too.

### CODE REVIEW

File: reference/constraints/Compound.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ you can create your own named set or requirements to be reused consistently ever
     // src/Validator/Constraints/PasswordRequirements.php
     namespace App\Validator\Constraints;
 
-    use Symfony\Component\Validator\Compound;
+    use Symfony\Component\Validator\Constraints\Compound;
```
Comment (wouterj) [inline reference/constraints/Compound.rst]: Can you remove this one from this PR and submit it to 5.1 instead?

File: translation.rst [hunk 2]
```diff
@@ -307,10 +307,10 @@ parts of your application and mocking it in your tests.
 
 Instead of translating a string at the time of creation, you can use a
 "translatable object", which is an instance of the
-:class:`Symfony\\Component\\Translation\\Translatable` class. This object stores
+:class:`Symfony\\Component\\Translation\\TranslatableMessage` class. This object stores
 all the information needed to fully translate its contents when needed::
 
-    use Symfony\Component\Translation\Translatable;
+    use Symfony\Component\Translation\TranslatableMessage;
```
Comment (wouterj) [inline translation.rst]: And can you maybe split these into a new PR as well? Then we can rebase this PR on 4.4 :)

**Review by wouterj — COMMENTED** [review]
Thank you checking all examples! Is it maybe possible for you to split the ones I commented on, so that each fix ends up in the correct branch? :grimacing:

File: reference/constraints/Compound.rst (hunk 1 above)
Comment (Nyholm) [reply reference/constraints/Compound.rst]: Done #14433

File: translation.rst (hunk 2 above)
Comment (Nyholm) [reply translation.rst]: This was fixed in #14371

**Comment by javiereguiluz** [comment]
Super nice fixes! Thanks Tobias.

File: translation.rst [hunk 3]
```diff
@@ -307,10 +307,10 @@ parts of your application and mocking it in your tests.
...
 "translatable object", which is an instance of the
-:class:`Symfony\\Component\\Translation\\TranslatableMessage` class. This object stores
+:class:`Symfony\\Component\\Translation\\Translatable` class. This object stores
 all the information needed to fully translate its contents when needed::
 
-    use Symfony\Component\Translation\TranslatableMessage;
+    use Symfony\Component\Translation\Translatable;
 
     // the first argument is required and it's the original message
     $message = new TranslatableMessage('Symfony is great!');
```
Comment (xabbuh) [inline translation.rst:316]: we need to change it here and below too

Comment (javiereguiluz) [reply translation.rst:316]: Good catch! Done in https://github.com/symfony/symfony-docs/commit/8d1364ecfb4db767168dcd33dcb045971ffc4ff6 Thanks!

**Pull request merged**